src/test/java/com/fasterxml/jackson/databind/jsontype/SerializablePolymorphicIdReaders4303Test.java

```diff
@@ -0,0 +1,72 @@
+package com.fasterxml.jackson.databind.jsontype;
+
+import java.io.ByteArrayOutputStream;
+import java.io.ObjectOutputStream;
+
+import org.junit.jupiter.api.Test;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+import com.fasterxml.jackson.databind.ObjectReader;
+
+import static com.fasterxml.jackson.databind.BaseMapTest.newJsonMapper;
+
+// [databind#4303]
+class SerializablePolymorphicIdReaders4303Test
```

Comment:
Instead of new test class, let's instead add into `com.fasterxml.jackson.databind.SerializeUsingJDKTest` .

Reply:
Is there class named `SerializeUsingJDKTest`? couldn't find any

Reply:
May have been renamed, let me check what it was named in 2.15.

Reply:
Ok yes, `TestJDKSerialization`. Renamed later on to be bit more descriptive.

Reply:
Found it, done via https://github.com/FasterXML/jackson-databind/pull/4304/commits/baf21e1b097316594a538b06f8785ca22a453c9e, thanks!